File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/rollout_check_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class RolloutCheckVO:
    """发布检查响应"""
    created_at: str = None
    updated_at: str = None
    rollout_key: str = None
    can_rollout: bool = None
    reason: str = None
    status: str = None
    checked_at: str = None

File: sdkwork-app-sdk-go/types/rollout_check_vo.go
package types

// 发布检查响应
type RolloutCheckVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	RolloutKey string `json:"rolloutKey"`
	CanRollout bool `json:"canRollout"`
	Reason string `json:"reason"`
	Status string `json:"status"`
	CheckedAt string `json:"checkedAt"`
}
